### PR TITLE
xGov-181-Research Article on Algorand Voting Technologies

### PR DESCRIPTION
**Pull request by Bhaney44**
xGov Grant Proposal.

### CODE REVIEW

**Comment by nonfungibleab** [comment]
HI @Bhaney44 please modify the title of your PR to match the expected format of:
xGov-PR#-Proposal title
Thanks, Adri

**Comment by nonfungibleab** [comment]
ie. xGov-181-Research Article on Algorand Voting Technologies

File: Proposals/xGov180.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+<a href = "https://forum.algorand.org/u/bhaney44">Brian Haney Algorand Forum</a>
+
+## Present Proposal
+- Research. (2 weeks) Collecting data, articles, and information relating to voting technologies and Algorand.
+- Outling. (1 week) Outlining the articles substance and identifying particular points and topics.
+- Drafting. (2 weeks) Writing the article.
+- Finalizing. (1 week) Reviewing, revising, and publishing the article.
+
+### Deliverable
+The deliverable will be a research article of at least 7,500 words with at least 75 sources. The sources will be professional, peer-reviewed, academic, and patent publications. The article will be published in an open source database and on GitHub.
```
Comment (nonfungibleab) [inline Proposals/xGov180.md:48]: Please provide public URL of where/how the community can check proposal deliverables.

Comment (Bhaney44) [reply Proposals/xGov180.md:48]: I can post deliverables and progress here: https://github.com/Bhaney44/Algorand-Voting-Research

**Pull request merged**